tests/golden_outputs/spark_t0_haploid_triploid_1_2_3_triploid_deletion_java_vcf

```diff
@@ -121,5 +121,5 @@
 1	12145	.	C	<NON_REF>	.	.	END=12277	GT:DP:GQ:MIN_DP:PL	.:2:0:0:0,0	.	./.:3:0:0:0,0,0	.
 1	12278	.	C	<NON_REF>	.	.	END=12295	GT:DP:GQ:MIN_DP:PL	.:2:0:0:0,0	.	.	.
 1	17384	.	TGG	T,TG,<NON_REF>	.	.	BaseQRankSum=1.07;ClippingRankSum=-1.067e+00;DP=9;MQ=55.72;MQ0=0;MQRankSum=0.056	GT:AD:DP:GQ:PL:SB	.	.	.	././.:6,2,3,0:8:40:40,0,197,58,203,262,123,456,789,12,0,1,2,3,4,5,6,7,8,9:0,0,0,0
-1	17385	.	G	A,T,*,<NON_REF>	.	.	BaseQRankSum=-2.074e+00;ClippingRankSum=-1.859e+00;DP=285;MQ=31.72;MQ0=3;MQRankSum=-4.320e-01;RAW_MQ=2.50;ReadPosRankSum=5.000e-03	GT:AD:DP:GQ:PGT:PID:PL:SB	././.:58,22,17,17,17:80:99:0|1:17385_G_A:504,0,9807,678,1870,2548,123,456,789,12,1870,2548,123,456,789,12,456,789,12,12,1870,2548,123,456,789,12,456,789,12,12,456,789,12,12,12:58,0,22,0	./.:40,36,0,0,0:76:99:.:.:1018,0,1116,1137,1224,2361,1137,1224,2361,2361,1137,1224,2361,2361,2361:9,31,13,23	./.:0,37,120,37,37:120:99:0|1:17385_G_T:3336,4536,7349,358,958,0,4536,7349,958,7349,4536,7349,958,7349,7349:0,0,0,0	././.:6,0,0,3,0:8:40:.:.:40,0,6,9,0,6,9,6,9,9,203,3,8,3,8,8,456,5,5,12,0,6,9,6,9,9,3,8,8,5,6,9,9,8,9:0,0,0,0
+1	17385	.	G	A,T,*,<NON_REF>	.	.	BaseQRankSum=-2.074e+00;ClippingRankSum=-1.859e+00;DP=285;MQ=31.72;MQ0=3;MQRankSum=-4.320e-01;RAW_MQ=2.50;ReadPosRankSum=5.000e-03	GT:AD:DP:GQ:PGT:PID:PL:SB	.|.|.:58,22,17,17,17:80:99:0|1:17385_G_A:504,0,9807,678,1870,2548,123,456,789,12,1870,2548,123,456,789,12,456,789,12,12,1870,2548,123,456,789,12,456,789,12,12,456,789,12,12,12:58,0,22,0	./.:40,36,0,0,0:76:99:.:.:1018,0,1116,1137,1224,2361,1137,1224,2361,2361,1137,1224,2361,2361,2361:9,31,13,23	./.:0,37,120,37,37:120:99:0|1:17385_G_T:3336,4536,7349,358,958,0,4536,7349,958,7349,4536,7349,958,7349,7349:0,0,0,0	././.:6,0,0,3,0:8:40:.:.:40,0,6,9,0,6,9,6,9,9,203,3,8,3,8,8,456,5,5,12,0,6,9,6,9,9,3,8,8,5,6,9,9,8,9:0,0,0,0
 1	17386	.	G	*,<NON_REF>	.	.	DP=9	GT:AD:DP:GQ:PL:SB	.	.	.	././.:6,3,0:8:40:40,203,456,12,0,3,5,6,8,9:0,0,0,0
```

Comment:
Just for my understanding, why is Java 11 causing phase changes? Is it because of a newer htsjdk? Is there a htsjdk option to preserve phase?

Reply:
Sorry, updating my comment after reading the Description for the PR. I see it is because of the newer htsjdk. But are the phase changes OK? See this [comment](https://github.com/GenomicsDB/GenomicsDB/pull/113#discussion_r482525986).

Reply:
I'll chime in here since Jaclyn and I discussed this a bit earlier. I think the changes are fine. If you keep reading the comments in the PR Jaclyn linked to, @kgururaj does give it the okay (https://github.com/GenomicsDB/GenomicsDB/pull/113#issuecomment-687158781). Briefly, htsjdk is limited in not allow different phase separators and they switched the behavior sometime between 2.10.0 and 2.23.0. This change in golden output allows to to hew to what they're expecting.

Reply:
Ah. Thanks, missed the comment there. This is good to go.